data-serving/data-service/src/model/demographics.ts

```diff
@@ -0,0 +1,41 @@
+import { Range } from './range';
+import mongoose from 'mongoose';
+
+export enum Sex {
+    Female = 'Female',
+    Male = 'Male',
+    Other = 'Other',
+}
+
+export enum Species {
+    HomoSapien = 'Homo sapien',
+}
+
+export const demographicsSchema = new mongoose.Schema({
+    ageRange: {
+        start: {
+            type: Number,
+            min: -1,
```

Comment:
Why would we include -1? And if we're constituting -1 as valid, shouldn't it be valid in `end`?

Reply:
-1 is for prenatal... I don't think there are any cases of this in the current line list but it was a suggestion from Tim to support it.

Fair that end should support it too, if there's a specific negative number in there.